src/index.ts

```diff
@@ -37,6 +37,12 @@ export interface PreactPluginOptions {
 	 */
 	prefreshEnabled?: boolean;
 
+	/**
+	 * Whether to use preact/compat aliases
+	 * @default true
+	 */
+	compatAliasesEnabled?: boolean;
```

Comment:
Alternatively, `preactCompatAliasesEnabled`? Not sure what's best, open to suggestions